docs/src/main/sphinx/functions/json.rst

```diff
@@ -6,12 +6,21 @@ Cast to JSON
 ------------
 
 Casting from ``BOOLEAN``, ``TINYINT``, ``SMALLINT``, ``INTEGER``,
-``BIGINT``, ``REAL``, ``DOUBLE`` or ``VARCHAR`` is supported.
-Casting from ``ARRAY``, ``MAP`` or ``ROW`` is supported when the element type of
-the array is one of the supported types, or when the key type of the map
+``BIGINT``, ``REAL``, ``DOUBLE``, or ``VARCHAR`` to JSON is supported.
+
+Casting from ``ARRAY``, ``MAP``, or ``ROW`` is supported when the element type of
+the array is one of the supported types, when the key type of the map
 is ``VARCHAR`` and value type of the map is one of the supported types,
 or when every field type of the row is one of the supported types.
```

Comment:
Would read better as a bulleted list IMO.

Reply:
Fixed, and cleaned up the following paragraph similarly